sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -4039,3 +4177,62 @@ def from_dict(cls, data):
             innererror=DocumentAnalysisInnerError.from_dict(data.get("innererror"))  # type: ignore
             if data.get("innererror") else None
         )
+
+def _get_children(source_element, search_elements, allowed_elements, cross_page=False):
```

Comment:
it seems like search_elements and allowed_elements could be collapsed into one param now that users don't specify the element_types. but let me know if I'm missing something

edit: If true, I suppose the check below could also be removed?